src/rpc/blockchain.cpp

```diff
@@ -52,10 +52,7 @@ static CUpdatedBlock latestblock;
  */
 double GetDifficulty(const CBlockIndex* blockindex)
 {
-    if (blockindex == nullptr)
-    {
-        return 1.0;
-    }
+    assert(blockindex);
```

Comment:
This call could fail the new assert:
https://github.com/bitcoin/bitcoin/blob/d6faea4f917efdbd6cefe6e91a61c101e20db434/src/rpc/mining.cpp#L217

Reply:
I believe `chainActive.Tip()` is always non null.